app/views/app_groups/_applications.html.slim

```diff
@@ -54,11 +54,13 @@ h4.mb-3 All Applications
           td.col-1= number_to_human_size(app.latest_ingested_log_bytes)
           td.col-1= sprintf(app.latest_cost.to_f < 10 ** 8 ? "%0.9f" : "%0.2f" ,app.latest_cost.to_f / 10 ** 9)
           td.col-1
+            .btn.btn-primary.btn-sm data-toggle="modal" data-target="#manage-label-modal" data-app-name=app.name data-app-path=update_labels_app_path(app)
```

Comment:
can we have spec test for this one? how you test the process? is it manually tested on your local?

Reply:
it is manually tested mas, as it is only print the URL path, not triggering any controller